serving/docs/out_of_memory_management.md

```diff
@@ -0,0 +1,21 @@
+# OutofMemory handling in djl-serving
```

Comment:
please also create a entry in the DJL main repo to refer to this markdown file link

Reply:
could you please point where?

Reply:
https://github.com/deepjavalibrary/djl/blob/master/docs/mkdocs.yml#L150